transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -1118,6 +1142,20 @@ private void callHandlerCallbackLater(AbstractChannelHandlerContext ctx, boolean
         }
     }
 
+    private void wakeUpForAutoFlush() {
+        if (autoFlush && ENQUEUE_WAKEUP_TASK_UPDATER.compareAndSet(this, 0, 1)) {
+            if (null == wakeupTask) {
+                wakeupTask = new Runnable() {
+                    @Override
+                    public void run() {
+                        ENQUEUE_WAKEUP_TASK_UPDATER.set(DefaultChannelPipeline.this, 0);
+                    }
+                };
+            }
+            channel().eventLoop().execute(wakeupTask);
+        }
+    }
+
```

Comment:
@jchambers how many connections do you have for this test?
There should only be a single `AutoFlushTask` in the queue per channel.